src/ansys/fluent/core/fluent_connection.py

```diff
@@ -782,8 +782,8 @@ def _exit(
             if cleanup_on_exit:
                 try:
                     connection_interface.exit_server()
-                except Exception:
-                    pass
+                except Exception as e:
+                    logger.debug(f"Failed to exit server: {e}")
```

Comment:
I'm not aware of any scenario where server is not exited, the try-except is to catch any potential grpc error while returning from the remote call from an exiting server.

Maybe a better logging message is `Error during exiting server`.

Reply:
Done. Thank you.